src/powerman/device_tcp.c

```diff
@@ -259,11 +255,8 @@ bool tcp_connect(Device * dev)
  */
 void tcp_disconnect(Device * dev)
 {
-    TcpDev *tcp;
-
     assert(dev->connect_state == DEV_CONNECTING
            || dev->connect_state == DEV_CONNECTED);
-    tcp = (TcpDev *)dev->data;
```

Comment:
this code removal doesn't seem related to the "reduce log noise" commit.  Seems like it's just dead code, meant for another commit and forgot it was in this one?

Reply:
Actually I changed some logging code and as a result, `tcp` became a set but not used error.  The full diff is
```diff
@ -259,11 +255,8 @@ bool tcp_connect(Device * dev)
  */
 void tcp_disconnect(Device * dev)
 {
-    TcpDev *tcp;
-
     assert(dev->connect_state == DEV_CONNECTING
            || dev->connect_state == DEV_CONNECTED);
-    tcp = (TcpDev *)dev->data;
 
     dbg(DBG_DEVICE, "tcp_disconnect: %s on fd %d", dev->name, dev->fd);
 
@@ -274,8 +267,7 @@ void tcp_disconnect(Device * dev)
         dev->fd = NO_FD;
     }
 
-    if (!tcp->quiet)
-        err(false, "tcp_disconnect(%s): disconnected", dev->name);
+    dbg(DBG_DEVICE, "tcp_disconnect(%s): disconnected", dev->name);
 }
 
 void tcp_preprocess(Device *dev)
```